source/guides/quick-start/vue-quick-start.md

```diff
@@ -0,0 +1,429 @@
+
+To capture screenshots, Cypress offers the command: `.screenshot()`
+
+Using the example of a login form, here is what the code might look like if we wanted to take a screenshot of the form before and after it is submitted.
+
+```jsx
+it('displays errors with incorrect credentials', () => {
+  cy.visit('/login')
+
+  cy.get('[data-cy="username"]').type('tony')
+  cy.get('[data-cy="password"]').type('infinity')
+
+  cy.screenshot()
+  cy.get('[data-cy="submit"').click()
+  cy.screenshot()
+})
+```
+
+## Debugging Tests
+
+Modern web applications are highly visual, interactive, and stateful thanks to frameworks like Vue.js that ease the creation of advanced UIs. As a result, Cypress enables workflows with helpful feedback loops for writing and running tests.
+
+For example, even though the ability to preview your app while Cypress is running is a convenience, there are times where the test goes to quickly for us to understand what happened.
+
+{% video local /img/guides/vue-quick-start/vue-quick-start-debugging-01.mp4 %}
+
+In this example, in the event we want to verify the state of our application, the test completes before we can register what happened. Fear not though, for Cypress' command log has another ability that we have not discussed yet: time travel.
+
+### Time Travel
+
+To help visually see the states of an app throughout a test, Cypress takes a DOM snapshot after each test command. This allows you to see how the state of the UI changed after every interaction or even toggle between how things looked at the time of a network request and at the moment of receiving a response. 
+
+In our example above, time traveling allows you to hover over individual steps in the testing process to see what is happening in your app.
+
+{% imgTag /img/guides/vue-quick-start/vue-quick-start-05.png "Screenshot of Cypress showing which element is hovered over" %}
+
+And if you click on the step, it will pin the DOM Snapshot so you can analyze it further.
+
+{% imgTag /img/guides/vue-quick-start/vue-quick-start-06.png "Screenshot of Cypress pinning a DOM snapshot" %}
+
+In addition, Cypress logs it to the console for further analysis!
+
+{% imgTag /img/guides/vue-quick-start/vue-quick-start-07.png "Screenshot of Cypress output when test step is clicked on" %}
+
+Here's a short clip of the time travel in action!
+
+{% video local /img/guides/vue-quick-start/vue-quick-start-debugging-02.mp4 %}
+
+With these automatic DOM snapshots, you'll have the {% url "ability to time-travel" test-runner#Hovering-on-Commands %} through the states of your UI for quick debugging or concisely understanding the behavior of your app.
+
+## Next Steps
+
+In this guide, we have covered the essentials you need for getting started, but there is much more in the world of Cypress.
+
+Thanks for reading our guide and hope it has helped you create more trust in your app.
```

Comment:
I can't explain why, but this sentence comes off a bit...odd. I think because it's more focused on our concerns of them trusting our app as opposed their concerns of what they get with Cypress. Maybe below is better?

```suggestion
Thanks for reading our guide and we hope if helps you successfully start testing your Vue applications.
```

Reply:
Agreed. This does read odd on another pass. How about we keep it short and simple?

`Thanks for reading our guide!`